### PR TITLE
a little bit of SEO

### PR DESCRIPTION
**Pull request by DrYak**
SEO through link-farming :

 - badge to framework used (snakemake)
 - badge to repositories listing (bio.tools, omictools)
 - badge with google-analytics beacon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 V-pipe is a workflow designed for clinical applications of next generation sequencing (NGS) data to viral pathogens. It produces a number of results in a curated format.
 
+[![Snakemake](https://img.shields.io/badge/snakemake-≥4.8.0-brightgreen.svg?style=plastic)](https://snakemake.bitbucket.io)
+[![bio.tools](https://img.shields.io/badge/bio-tools-orange.svg?style=plastic)](https://bio.tools/V-Pipe)
```
Comment (sposadac) [inline README.md]: Can you change the contact information to the V-pipe mail?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+[![omic.tools](https://img.shields.io/badge/omic-tools-blue.svg?style=plastic)](https://omictools.com/v-pipe-tool)
+[![License: MIT](https://img.shields.io/badge/License-Apache_2.0-yellow.svg?style=plastic)](https://opensource.org/licenses/Apache-2.0)
```
Comment (sposadac) [inline README.md]: We use the Apache 2.0 licence, not the MIT License

File: README.md (hunk 1 above)
Comment (DrYak) [reply README.md]: modifications force-pushed

File: README.md (hunk 2 above)
Comment (DrYak) [reply README.md]: thanks for spotting my error

File: README.md (hunk 1 above)
Comment (sposadac) [reply README.md]: The two comments above were concerning the bio.tools website

**Pull request merged**